alarm/models.py

```diff
@@ -25,12 +25,12 @@ class Threshold(models.Model):
     price = models.DecimalField(max_digits=10, decimal_places=2)
 
     def __str__(self):
-        return f"{self.trade_pair}: {self.price}"
+        return f"{self.price}"
 
     def clean(self):
         super().clean()
-        valid_trade_pair = get_binance_trade_pairs()
-        if self.trade_pair not in valid_trade_pair:
+        valid_list_of_coin_names = get_binance_valid_list_of_coin_names()
+        if self.trade_pair not in valid_list_of_coin_names:
```

Comment:
Inconsistency here. You first call it trade pair in self.trade pair, then coin names.
Just revert.

Reply:
++